Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -382,6 +382,9 @@ ModelExperimentalSceneGraph.prototype.buildDrawCommands = function (
   const model = this._model;
   const modelRenderResources = new ModelRenderResources(model);
 
+  // Reset the memory counts before running the pipeline
+  model.statistics.clear();
```

Comment:
Hm for the tileset test, double check that the test isn't resetting the pipeline at an unexpected time. I doubt it, but better safe to check.